### ISSUE TITLE
Inline edit of records in all subpanels

### ISSUE THREAD

**Issue by Mausino** [salesagility/SuiteCRM#1043]
He,

In old roadmap was, that the records will possible inline edit in subpanels.
I think that will so great,  if it will realize, because it saves so much time :)
- if we have more and more records in subpanels, it also hepls this #501 Asynchronous subpanels 

Miky

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Inline edit of records in all subpanels](https://trello.com/c/BFR496TU/115-inline-edit-of-records-in-all-subpanels)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
